src/Avalonia.Visuals/Media/FormattedText.cs

```diff
@@ -10,128 +10,109 @@ namespace Avalonia.Media
     /// <summary>
     /// Represents a piece of text with formatting.
     /// </summary>
-    public class FormattedText : AvaloniaDisposable
+    public class FormattedText
```

Comment:
Well there is the [GC.AddMemoryPressure](https://msdn.microsoft.com/en-us/library/system.gc.addmemorypressure(v=vs.110).aspx) method, but I agree we'll probably want to be able to dispose of these objects at some point. I'd prefer to do that in another PR however, as I'd like to get the scenegraph stuff ported over this year ;)